phalcon/Config/ConfigFactory.zep

```diff
@@ -97,11 +97,11 @@ class ConfigFactory extends AbstractFactory
             );
         }
 
-        let adapter = strtolower(config["adapter"]),
+        let adapter = mb_strtolower(config["adapter"]),
```

Comment:
Is there any reason to use multi byte function for the adapter name? Do we plan provide support for PHP non in ASCII? AFAIK, mbstring functions are slower. So, it is curious to me, what we offer for such the price? Adapter names in national encodings?

Reply:
I don't mind paying where it is necessary. But I don't quite understand why I should always pay for what I don't use.